Log file created at: 2020/05/25 17:46:41
Running on machine: ai07
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0525 17:46:41.672133 151200 util.h:71] NewGlobal N7oneflow7EnvDescE
I0525 17:46:41.675861 151200 util.h:71] NewGlobal N7oneflow10CtrlServerE
I0525 17:46:41.686455 151200 ctrl_server.cpp:39] CtrlServer listening on 0.0.0.0:9789
I0525 17:46:41.686542 151200 util.h:71] NewGlobal N7oneflow10CtrlClientE
I0525 17:46:41.687335 151200 ctrl_client.cpp:187] LoadServer 127.0.0.1 Successful at 0 times
I0525 17:46:41.687391 151200 util.h:71] NewGlobal N7oneflow10MachineCtxE
I0525 17:46:41.687407 151200 machine_context.cpp:16] this machine id: 0
I0525 17:46:41.703415 151200 util.h:71] NewGlobal N7oneflow12ResourceDescE
I0525 17:46:41.703431 151200 util.h:71] NewGlobal N7oneflow6IOConfE
I0525 17:46:41.703436 151200 util.h:71] NewGlobal N7oneflow12ProfilerConfE
I0525 17:46:41.703441 151200 util.h:71] NewGlobal N7oneflow5IDMgrE
I0525 17:46:41.887956 151200 util.h:71] NewGlobal N7oneflow16AvailableMemDescE
I0525 17:46:41.888140 151200 util.h:71] NewGlobal St13unordered_mapISslSt4hashISsESt8equal_toISsESaISt4pairIKSslEEE
I0525 17:46:41.888149 151200 util.h:71] NewGlobal N7oneflow19CriticalSectionDescE
I0525 17:46:41.888154 151200 util.h:71] NewGlobal N7oneflow16InterUserJobInfoE
I0525 17:46:41.888160 151200 util.h:71] NewGlobal N7oneflow22JobBuildAndInferCtxMgrE
I0525 17:46:41.888164 151200 util.h:71] NewGlobal N7oneflow18LbiDiffWatcherInfoE
I0525 17:46:41.888170 151200 util.h:71] NewGlobal N7oneflow16JobSetCompileCtxE
I0525 17:46:41.888175 151200 util.h:71] NewGlobal N7oneflow26RuntimeBufferManagersScopeE
I0525 17:46:41.888180 151200 util.h:71] NewGlobal N7oneflow9BufferMgrIlEE
I0525 17:46:41.888182 151200 util.h:71] NewGlobal N7oneflow9BufferMgrISt10shared_ptrINS_18ForeignJobInstanceEEEE
E0525 17:46:41.888533 151200 error.cpp:11] /workspace/oneflow/core/framework/load_library.cpp:9
 Check failed: handle	 LoadLibrary ERROR! Cannot load library file: /home/zjlab/anaconda3/envs/oneflow/lib/python3.5/site-packages/oneflow_yolov3/liboneflow_yolov3.so the Error is: libcudart.so.10.0: cannot open shared object file: No such file or directory
I0525 17:46:41.895704 151200 session_global_objects_scope.cpp:76] maybe failed:/workspace/oneflow/core/job/session_global_objects_scope.cpp:76
I0525 17:46:41.895710 151200 oneflow_internal_helper.h:78] maybe failed:/workspace/oneflow/python/oneflow_internal_helper.h:78
